### PR TITLE
Added Arrays.cpp, Arrays.h and HW9.cpp files.

### PR DESCRIPTION
**Pull request by shintanshin**
Please review my 9th homework.

### CODE REVIEW

File: HW9/Arrays.cpp [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    }
+    for (int i = 0; i < size; i++) {
+        (numbers[i] > 0) ? numbers[i] *= 2 : numbers[i] = 0;
+    }
+    std::cout << "Array elements are:" << std::endl;
+    for (int i = 0; i < size; i++) {
+        std::cout << numbers[i] << " ";
+    }
+    std::cout << std::endl;
+}
```
Comment (bohdanpc) [inline HW9/Arrays.cpp]: It shouldn't input, or output arrays. Just translate an array passed through arg.

File: HW9/Arrays.cpp [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+void toUppercase(char str[]) {
+    const int a = 96;
```
Comment (bohdanpc) [inline HW9/Arrays.cpp]: alphaSymbolStart - alphaSymbolEnd
start - end
alphaStart - alphaEnd
firstSymbol - lastSymbol
or similar but with context not just a - b

File: HW9/Arrays.cpp [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    const int b = 123;
+    std::cout << "Input: ";
+    std::cin >> str;
```
Comment (bohdanpc) [inline HW9/Arrays.cpp]: same: no input/output

File: HW9/Arrays.cpp [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    for (int i = 0; str[i] != '\0'; i++) {
+        if (str[i] > a && str[i] < b) { str[i] = toupper(str[i]); }
+    }
+    std::cout << "Output: " << str << std::endl;
+}
+
+bool isPalindrom(const char str[]) {
+    int value = 0;
```
Comment (bohdanpc) [inline HW9/Arrays.cpp:36]: bool

File: HW9/Arrays.cpp [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            }
+        }
+    }
+}
+bool isEqual(const char str1[], const char str2[]) {
+    int i = 0;
+    int value = 0;
+    while (str1[i] != '\0' || str2[i] != '\0') {
+        if (str1[i] != str2[i]) {
+            value = 1;
```
Comment (bohdanpc) [inline HW9/Arrays.cpp:74]: return false;

File: HW9/Arrays.cpp [hunk 6]
```diff
@@ -0,0 +1,82 @@
+﻿#include <iostream>
+
```
Comment (bohdanpc) [inline HW9/Arrays.cpp:2]: In general, none of these functions are intended to read input or write input.

**Review by helen-black — APPROVED** [review]

**Pull request merged**